src/robusta/core/playbooks/common.py

```diff
@@ -16,7 +16,10 @@ def get_resource_events_table(table_name: str, kind: str, name: str, namespace:
     if event_list.items:
         headers = ["reason", "type", "time", "message"]
         rows = [
-            [event.reason, event.type, parse_kubernetes_datetime_to_ms(event.lastTimestamp), event.message]
+            [event.reason,
+             event.type,
+             parse_kubernetes_datetime_to_ms(get_event_timestamp(event)) if get_event_timestamp(event) else "Missing Timestamp",
```

Comment:
Did you check this is parsed properly on all sinks?
This column type has a DATETIME renderer, which means the UI (and maybe other sinks) will try to render it as timestamp.
We need to make sure it doesn't break.

Reply:
<img width="1141" alt="Screen Shot 2022-03-09 at 15 05 04" src="https://user-images.githubusercontent.com/29005743/157447405-cb6a8213-312e-4933-b0f2-758a5eba6ec6.png">
in the ui when the string is "Missing timestamp"

Reply:
Checkout render_value function (used by other sinks). This is used for rendering tables as strings in Slack etc
Does it fails nicely here as well?

def render_value(renderer: RendererType, value):
    if renderer == RendererType.DATETIME:
        date_value = datetime.fromtimestamp(value / 1000.0)
        return date_value.astimezone(DEFAULT_TIMEZONE).strftime(
            "%b %d, %Y, %I:%M:%S %p"
        )
    raise Exception(f"Unsupported renderer type {renderer}")

Reply:
i got it working with 0